### ISSUE TITLE
Screen Reader: Global: Informative dot icon near bottom tabs is not announced

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77362]
## Action Performed:
Precondition: User is logged in

1. Using VoiceOver on iOS or TalkBack Android, open the app
2. On the Inbox screen, swipe right to focus the Account bottom bar and note if the informative dot near it is announced

This is a global issue occurring in all bottom navigation bar tabs

## Expected Result:
Screen readers should announce the informative dot icon’s meaning, e.g.:

Example announcements:
"Account, button. Actions required"
"Inbox, button. Actions required"

## Actual Result:
Screen reader users cannot perceive the presence or meaning of the informative icon.

## Area issue was found in:
Inbox

## Failed WCAG checkpoints
WCAG 1.1.1 Non-Text Content
https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html

## User impact:
Users who have difficulty perceiving visual content use assistive technology to read text aloud, present it visually, or convert it to braille. Text alternatives may help some people who have difficulty understanding the meaning of photographs, drawings, and other images (e.g., line drawings, graphic designs, paintings, three-dimensional representations), graphs, charts, animations, etc. People who are deaf-blind can read the text alternatives in braille.

## Suggested resolution:
Include the informative dot’s meaning in the tab’s accessibilityLabel to ensure it is communicated with it, for example:
"Account, button. Actions required"
"Inbox, button. Actions required"
https://appt.org/en/docs/react-native/samples/accessibility-label
https://reactnative.dev/docs/accessibility#accessibilitylabel

## Environment(s)
Apple iPhone 14 Pro - iOS 18.7.1, Wi-Fi - Native (No mobile browser)
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
![Image](https://github.com/user-attachments/assets/0f5aaea1-97f2-4492-948a-d65f35289474)

https://github.com/user-attachments/assets/31129a11-768d-4ad0-b8b5-0074e381082f

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7005824
**Issue reported by:** NA
**Slack conversation:** NA
